Source/Tools/FEXLoader/LinuxSyscalls/GdbServer.cpp

```diff
@@ -58,6 +58,59 @@ desc: Provides a gdb interface to the guest state
 
 namespace FEX
 {
+
+constexpr std::array<std::string_view const, 22> FlagNames = {
+  "CF",
+  "",
+  "PF",
+  "",
+  "AF",
+  "",
+  "ZF",
+  "SF",
+  "TF",
+  "IF",
+  "DF",
+  "OF",
+  "IOPL",
+  "",
+  "NT",
+  "",
+  "RF",
+  "VM",
+  "AC",
+  "VIF",
+  "VIP",
+  "ID",
+};
+
+static std::string_view const& GetFlagName(unsigned Flag) {
+  return FlagNames[Flag];
+}
+
+constexpr std::array<std::string_view const, 16> RegNames = {
+  "rax",
+  "rcx",
```

Comment:
How is this ordered now? The PR description isn't specific enough for me to just find it, so sounds like something that needs a comment here.

Reply:
Changed over to a switch statement so that even if the enum ordering changes in the future this won't break again. Since it is highly likely to change in the future. This way the ordering matching doesn't matter anymore.